changelog/_unreleased/2025-03-14-add-acl-hook-service.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Add acl hook service
+author: Aydin Hassan
+---
+# Core
+* Added an `acl` service available in all hook scripts to check if your app has been granted a given privilege.
```

Comment:
‼️ The new facade needs to be added to `\Shopware\Core\Framework\Script\ServiceStubs`